dandiapi/zarr/tests/factories.py

```diff
@@ -28,3 +35,34 @@ def ensure_draft_version(obj: ZarrArchive, *args, **kwargs):  # type: ignore  #
 
 class EmbargoedZarrArchiveFactory(ZarrArchiveFactory):
     dandiset = factory.SubFactory(DandisetFactory, embargo_status=Dandiset.EmbargoStatus.EMBARGOED)
+
+
+class ZarrFileFactory(PostgenerationAttributesFactory):
+    class Meta:
+        model = ZarrArchiveFile
+
+    class Params:
+        blob = factory.Faker('binary', length=100)
+        zarr_archive = factory.SubFactory(ZarrArchiveFactory)
+
+    path = factory.Faker('file_path', absolute=False, extension='nwb')
```

Comment:
Due to the structure of zarrs, a realistic path for a zarr file (relative to the zarr root) would be something like 

```
1/0/10/10
```

I think to achieve this behavior, we'd probably need to create a custom faker provider. If nothing else though, I think we should just go with no file extension.

```suggestion
    path = factory.Faker('file_path', absolute=False, extension='')
```